logging/interface.go

```diff
@@ -0,0 +1,43 @@
+// Package logging defines a logging interface for quic-go.
+// This package should not be considered stable
+package logging
+
+import (
+	"net"
+	"time"
+
+	"github.com/lucas-clemente/quic-go/internal/congestion"
+	"github.com/lucas-clemente/quic-go/internal/protocol"
+	"github.com/lucas-clemente/quic-go/internal/wire"
+)
+
+type Tracer interface {
+	TracerForServer(odcid protocol.ConnectionID) ConnectionTracer
+	TracerForClient(odcid protocol.ConnectionID) ConnectionTracer
+}
```

Comment:
We'll need to add another method here for logging events not related to any particular connection (e.g. stateless resets sent, Version Negotiation packets sent, packets that get dropped before they are sent to a connection, etc.).
Any suggestion for a name?

Reply:
Why a single method, and not one per event, like in the ConnectionTracer?

Reply:
That makes sense. Good thing about it, we don't need to come up with another name (`NonConnectionTracker`?).